src/main/frontend/src/app/layout/sidebars/sidebars.component.ts

```diff
@@ -0,0 +1,42 @@
+import { AfterViewInit, Component, Input, OnDestroy, viewChild } from '@angular/core';
+import { IconComponent } from '../../components/icon/icon.component';
+import { SplitAreaComponent, SplitComponent, SplitGutterDirective } from 'angular-split';
+import { NgClass } from '@angular/common';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'app-sidebars',
+  imports: [IconComponent, SplitAreaComponent, SplitComponent, NgClass, SplitGutterDirective],
+  templateUrl: './sidebars.component.html',
+  styleUrl: './sidebars.component.scss',
+})
+export class SidebarsComponent implements AfterViewInit, OnDestroy {
+  @Input() leftCollapsed = false;
+  @Input() rightCollapsed = false;
+  protected readonly window = globalThis;
```

Comment:
`window` does not seem to be used anywhere since `globalThis` is still used everywhere